src/main/java/uk/gov/ons/ctp/response/collection/exercise/validation/ValidationScheduler.java

```diff
@@ -12,7 +12,12 @@
 @Component
 public class ValidationScheduler {
 
-  @Autowired private SampleService sampleService;
+  private SampleService sampleService;
+
+  @Autowired
+  public ValidationScheduler(SampleService sampleService) {
```

Comment:
you don't even have to annotate constructors with Autowired if you don't want to